contracts/account/extensions/AccountERC7579.sol

```diff
@@ -40,7 +40,16 @@ import {Account} from "../Account.sol";
  *   internal virtual functions {_extractUserOpValidator} and {_extractSignatureValidator}. Both are implemented
  *   following common practices. However, this part is not standardized in ERC-7579 (or in any follow-up ERC). Some
  *   accounts may want to override these internal functions.
+ * * When combined with {ERC7739}, resolution ordering of {isValidSignature} may have an impact ({ERC7739} does not
+ *   call super). Manual resolution might be necessary.
+ * * Static calls (using callType 0xfe) are currently NOT supported.
  * ====
+ *
+ * [WARNING]
+ * ===
+ * * When uninstalling validation module, users should make sure to keep at least one validation mechanism active.
+ *   Uninstalling all the validation mechanisms would brick the account.
+ * ===
```

Comment:
I think this can be simplified to just

```suggestion
 * WARNING: Removing all validator modules will render the account inoperable, as no user operations can be validated thereafter.
```

Note I'm purposely not using the term "brick" since I think it may be confusing.